### PR TITLE
PR: Fix tests

### PR DESCRIPTION
**Pull request by goanpeca**
Making sure all tests run :-)

### CODE REVIEW

File: spyder_unittest/widgets/tests/test_unittestgui.py [hunk 1]
```diff
@@ -33,7 +31,8 @@ def test_run_tests_and_display_results(qtbot, tmpdir):
     qtbot.wait(1000) # wait for tests to run
```
Comment (goanpeca) [inline spyder_unittest/widgets/tests/test_unittestgui.py]: @jitseniesen in general I dont think this is a good idea. The better way to do this is with a signal and using this construct

http://pytest-qt.readthedocs.io/en/latest/signals.html

```python
    with qtbot.waitSignal(app.worker.finished, timeout=10000) as blocker:
        app.worker.start()
        # Test will block at this point until either the "finished" or the
        # "failed" signal is emitted. If 10 seconds passed without a signal,
        # SignalTimeoutError will be raised.

```

Comment (jitseniesen) [inline spyder_unittest/widgets/tests/test_unittestgui.py]: Agreed --> issue #35

Comment (goanpeca) [inline spyder_unittest/widgets/tests/test_unittestgui.py]: I think this is the reason why tests are failing on qt4.

This PR is already enforcing test passing, so for now all fails :-(

**Comment by goanpeca** [comment]
@jitseniesen so circleci (linux) passes, osx and windows do not. Windows hangs and OSX fails the test. Do you want me to merge as it is so that you can check fixing it on a subsequent PR?

**Comment by jitseniesen** [comment]
@goanpeca Thanks. I never ran the test on Windows and OSX so it's not surprising that they fail.

**Pull request merged**